.github/workflows/cicd-main.yml

```diff
@@ -17,7 +17,7 @@ on:
     - cron: 0 0 * * *
   push:
     branches:
-      - main
+      - ray-api
```

Comment:
Should this be main now

Reply:
Updating here: https://github.com/NVIDIA-NeMo/Curator/pull/962.